src/shared/components/Snackbar/Snackbar.style.ts

```diff
@@ -1,72 +1,111 @@
+    height: 0;
+    margin-bottom: 0;
   }
-  .snackbar-enter-active {
-    transition: transform ${transitions.timings.loading} ${transitions.easing};
+
+  &.snackbar-enter-active {
+    transition: height ${transitions.timings.regular} ${transitions.easing},
+      margin-bottom ${transitions.timings.regular} ${transitions.easing},
+      transform ${transitions.timings.regular} ${transitions.easing} ${transitions.timings.regular};
   }
-  .snackbar-exit-active {
-    transition: transform 800ms ${transitions.easing};
+  &.snackbar-exit-active {
+    transition: transform ${transitions.timings.regular} ${transitions.easing},
+      height ${transitions.timings.regular} ${transitions.easing} ${transitions.timings.regular},
+      margin-bottom ${transitions.timings.regular} ${transitions.easing} ${transitions.timings.regular};
   }
 `
 
-export const SnackbarWrapper = styled.div`
-  background-color: ${colors.gray[800]};
+export const SnackbarHeader = styled.div`
   display: flex;
-  justify-content: space-between;
   align-items: center;
-  padding: ${sizes(2)};
-  padding-left: ${sizes(4)};
-  max-width: 300px;
   width: 100%;
-  position: fixed;
-  bottom: ${SNACKBAR_POSITION_MOBILE}px;
-  left: ${SNACKBAR_POSITION_MOBILE}px;
-  z-index: ${zIndex.overlay};
-  margin: 0;
-  @media screen and (min-width: ${breakpoints.small}) {
-    padding: ${sizes(3)};
-    padding-left: ${sizes(6)};
-    max-width: 400px;
-    bottom: ${SNACKBAR_POSITION}px;
-    left: ${SNACKBAR_POSITION}px;
+`
+
+export const StyledInnerWrapper = styled.div<InnerWrapperProps>`
+  width: 100%;
+  color: ${({ variant }) => (variant === 'secondary' ? colors.gray[300] : colors.blue[100])};
+  padding: ${({ hasSubMessage }) => (hasSubMessage ? `${sizes(4)} ${sizes(5)}` : `${sizes(3)} ${sizes(5)}`)};
+  width: 100%;
+  ${SnackbarHeader} {
+    color: ${({ hasSubMessage }) => hasSubMessage && colors.white};
   }
 `
 
-export const SnackbarButton = styled.button`
-  border: none;
-  background: none;
-  color: ${colors.gray[300]};
-  padding: ${sizes(3)};
-  cursor: pointer;
+export const SnackbarButtonsContainer = styled.div`
+  display: flex;
+  margin-left: auto;
 `
 
-export const SnackbarParagraph = styled.p`
-  margin: 0;
-  font-size: ${typography.sizes.body2};
-  color: ${colors.white};
+export const SnackbarExitButton = styled(Button)`
+  &:hover {
+    background-color: ${colors.transparent};
+  }
+`
+
+export const SnackbarActionButton = styled(Button)`
   display: flex;
   align-items: center;
-  word-break: break-all;
-  svg {
-    margin-right: ${sizes(4)};
-    width: ${sizes(6)};
-    height: ${sizes(6)};
+  padding: 0;
+  min-width: auto;
+  font-size: ${sizes(3)};
+  margin-right: ${sizes(2)};
+  font-size: ${typography.sizes.body1};
+  span {
+    margin-top: calc(-1 * ${sizes(1)});
+  }
+  &:hover {
+    background-color: ${colors.transparent};
   }
 `
+
+export const SnackbarDescription = styled(Text)`
+  margin-top: ${sizes(1)};
```

Comment:
Seem there's some weird style applied to this in Figma. Let's use the following here:
`padding: ${sizes(1)} 0;` instead of `margin-top`. This should bring us close to height specified in Figma. Then if button is present as well we can use `margin-bottom: ${sizes(3)};`

Reply:
have to add color here after rebase